sycl/ReleaseNotes.md

```diff
@@ -16,6 +16,699 @@
 
 - Did this and that ... intel/llvm#pr
 
+# Release notes Mar'25
+
+Release notes for commit range
+[b0212c37b2](https://github.com/intel/llvm/commit/b0212c37b230d9dd3bb129df9f4ecc417b92ad8)
+...
+[b23d69e2c3](https://github.com/intel/llvm/commit/b23d69e2c3fda1d69351137991897c96bf6a586d)
+
+## New Features
+
+### Runtime compilation of SYCL code
+
+- [`sycl_ext_oneapi_kernel_compiler`](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler.asciidoc)
+  extension specification was updated to accept `sycl` as source language, thus
+  providing functionality similar to
+  [NVRTC](https://docs.nvidia.com/cuda/nvrtc/). intel/llvm#11985,
+  intel/llvm#17446
+- Initial support for this feature was implemented. intel/llvm#16132,
+  intel/llvm#16222, intel/llvm#16132, intel/llvm#17640, intel/llvm#17356,
+  intel/llvm#16565, intel/llvm#17383, intel/llvm#17447, intel/llvm#17307,
+  intel/llvm#17373, intel/llvm#17331, intel/llvm#17329, intel/llvm#17266,
+  intel/llvm#17032, intel/llvm#16823, intel/llvm#16702, intel/llvm#16638,
+  intel/llvm#16316, intel/llvm#17359, intel/llvm#16485, intel/llvm#16821
+- Known issues and limitations are documented
+  [in the extension specification](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler.asciidoc#non-normative-implementation-notes-for-dpc). intel/llvm#17307,
+  intel/llvm#17459
+
+### SYCL graphs
+
+- Introduced and implemented
+  [`sycl_ext_codeplay_enqueue_native_command`](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/experimental/sycl_ext_codeplay_enqueue_native_command.asciidoc)
+  extension which allows to include custom commands for interoperability with
+  native runtimes into graphs built using `sycl_ext_oneapi_graph` extension.
+  intel/llvm#16871
+
+### Bindless images
+
+- Extended the extension
+  [specification](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc)
+  to support more kinds of copy operations (`image_mem_handle` to USM and vice
+  versa, USM to USM, etc.) and implemented them. intel/llvm#16661,
+  intel/llvm#17507
+- Extended the extension specification and implementation to support
+  `gather_image` device built-in function. The implementation was only done for
+  CUDA backend so far. intel/llvm#17322
+
+### Native CPU Device
+
+- Added support for source-based code coverage on Native CPU. intel/llvm#15073
+
+### KHR extensions
+
+Please note that KHR extensions are being specified and released by Khronos
```

Comment:
@gmlueck, tagging you here for feedback. I'm on a verge about whether or not we should mention implementation of unfinished KHRs in our release notes. The proposed wording is to do document them to share awareness/give some insights, but at the same time they may not be ready to be used yet.

Reply:
@gmlueck, could you please comment?

Reply:
I'd say that we should not mention the "unfinished" KHR extensions in the release notes.  It seems like they are not really safe to use because they aren't finished -- they may not implement all parts of the spec, may have bugs, etc.  The text below says that advanced users driving the development of these KHRs might want to use the current implementation.  However, these people already know about the implementation, so there is no need to mention it in the release notes.

Note that the KHR "sycl_khr_default_context" listed below is easy to miss.  That KHR *is* finished, and it should be mentioned in the release notes.  However, we list it after all the paragraphs about the unsafe "unfinished" KHRs, so it's easy to miss or to think that this KHR is also unfinished.

Reply:
@gmlueck thanks, removed. Could you please check one more time?

Reply:
Looks good!